lte/gateway/c/oai/tasks/nas/emm/sap/emm_recv.c

```diff
@@ -277,18 +277,18 @@ int emm_recv_attach_request(
 
   if (msg->presencemask & ATTACH_REQUEST_LAST_VISITED_REGISTERED_TAI_PRESENT) {
     params->last_visited_registered_tai = calloc(1, sizeof(tai_t));
-    params->last_visited_registered_tai->mcc_digit1 =
-        msg->lastvisitedregisteredtai.mcc_digit1;
-    params->last_visited_registered_tai->mcc_digit2 =
-        msg->lastvisitedregisteredtai.mcc_digit2;
-    params->last_visited_registered_tai->mcc_digit3 =
-        msg->lastvisitedregisteredtai.mcc_digit3;
-    params->last_visited_registered_tai->mnc_digit1 =
-        msg->lastvisitedregisteredtai.mnc_digit1;
-    params->last_visited_registered_tai->mnc_digit2 =
-        msg->lastvisitedregisteredtai.mnc_digit2;
-    params->last_visited_registered_tai->mnc_digit3 =
-        msg->lastvisitedregisteredtai.mnc_digit3;
+    params->last_visited_registered_tai->plmn.mcc_digit1 =
+        msg->lastvisitedregisteredtai.plmn.mcc_digit1;
+    params->last_visited_registered_tai->plmn.mcc_digit2 =
+        msg->lastvisitedregisteredtai.plmn.mcc_digit2;
```

Comment:
Generic comment to use existing macros, COPY_PLMN and COPY_TAI while plmn and tai accordingly

Reply:
Resolved